tests/test_annotator.py

```diff
@@ -17,8 +17,10 @@ class TestAnnotator(unittest.TestCase):
 
     def setUp(self):
         self.data = [[1, 2, 3], [2, 5, 7]]
-        self.data2 = pd.DataFrame([[1, 2], [2, 5], [3, 7]], columns=["X", "Y"])
         self.ax = sns.boxplot(data=self.data)
+
+        self.data2 = pd.DataFrame([[1, 2], [2, 5], [3, 7]], columns=["X", "Y"])
+
```

Comment:
Would you please move data2 above where it was? (You can compare the last commit before yours with the last to see the sum of all changes.)